### PR TITLE
Make `zephir clean` and `fullclean` work in Windows

### PR DESCRIPTION
**Pull request by sjinks**
See #701

### CODE REVIEW

File: Library/Compiler.php [hunk 1]
```diff
@@ -1260,7 +1260,12 @@ public function test(CommandInterface $command)
      */
     public function clean(CommandInterface $command)
     {
-        system('cd ext && make clean 1> /dev/null');
+        $this->fileSystem->clean();
+        if (Utils::isWindows()) {
+            system('cd ext && nmake clean-all');
+        } else {
+            system('cd ext && make clean > /dev/null');
+        }
```
Comment (sergeyklay) [inline Library/Compiler.php:1268]: @sjinks Could we add something like this: https://github.com/phalcon/php-zephir-parser/blob/development/unit-tests/Extension/TestCase.php#L54-L77

related to the https://github.com/phalcon/zephir/issues/1462

Comment (sergeyklay) [reply Library/Compiler.php:1268]: https://github.com/phalcon/zephir/blob/master/Library/FileSystem/HardDisk.php#L166

Comment (sjinks) [reply Library/Compiler.php:1268]: Please take a look

File: .gitignore [hunk 2]
```diff
@@ -94,3 +94,10 @@ parser/parser.out
 parser/Release
 parser/lemon
 parser/lemon.exe
+
+# Eclipse
+.cproject
+.project
+.settings/
+.buildpath
+.tracefile
```
Comment (sergeyklay) [inline .gitignore]: What about https://help.github.com/articles/ignoring-files/#create-a-global-gitignore 

See top of this file :)

**Comment by sergeyklay** [comment]
Thank you!

**Pull request merged**